sdk/compute/mgmt/src/main/java/com/azure/resourcemanager/compute/implementation/SnapshotImpl.java

```diff
@@ -335,7 +335,7 @@ private String constructStorageAccountId(String vhdUrl) {
                 "storageAccounts",
                 vhdUrl.split("\\.")[0].replace("https://", ""),
```

Comment:
Not sure whether it may be `http` or not. I think it better to extract `host` from `url` first.